tutorials/create-cloud-build-image-factory-using-packer/index.md

```diff
@@ -41,59 +59,297 @@ gcloud config set compute/zone $ZONE
+
+Invoke-WebRequest -Uri "https://github.com/GoogleCloudPlatform/community/raw/master/tutorials/create-cloud-build-image-factory-using-packer/windows/scripts/install-chocolatey.ps1" -OutFile "./scripts/install-chocolatey.ps1"
+
+Invoke-WebRequest -Uri "https://github.com/GoogleCloudPlatform/community/raw/master/tutorials/create-cloud-build-image-factory-using-packer/windows/scripts/run-chocolatey.ps1" -OutFile "./scripts/run-chocolatey.ps1"
+```
+
+</details>
+
+3.  Initialize a git repository in the working directory:
+
+<details>
+<summary>Click to reveal commands</summary>
+
+```sh
+git init
+```
+
+</details>
+
+# Task 3: Enable the required services
+Enable the Google Cloud APIs necessary for the tutorial
+
+## Building a Linux Image
+
+<details>
+<summary>Click to reveal commands</summary>
+
+```sh
+gcloud services enable sourcerepo.googleapis.com
+gcloud services enable cloudapis.googleapis.com
+gcloud services enable compute.googleapis.com
+gcloud services enable servicemanagement.googleapis.com
+gcloud services enable storage-api.googleapis.com
+gcloud services enable cloudbuild.googleapis.com
+```
+</details>
+
+## Building a Windows Image
+
+<details>
+<summary>Click to reveal commands</summary>
+
+```sh
+gcloud services enable sourcerepo.googleapis.com
+gcloud services enable cloudapis.googleapis.com
+gcloud services enable compute.googleapis.com
+gcloud services enable servicemanagement.googleapis.com
+gcloud services enable storage-api.googleapis.com
+gcloud services enable cloudbuild.googleapis.com
+gcloud services enable secretmanager.googleapis.com
+```
+</details>
+
+# Task 3a: Managing secrets for params using the Google Secrets Manager (Windows Image Only)
+In this section, we will be using the [Google Secrets Manager](https://cloud.google.com/secret-manager) to store our input values for packer in a secure and modular way. Although it's easier to simply hard code parameters into the packer template file (eg. network tags, source GCE image), using a central source of truth like a secrets manager increases manageability and reuseability amongst teams.
+
+Create your secrets using the following commands. Optionally, you can customize the values using the [documentation](https://cloud.google.com/secret-manager/docs/creating-and-accessing-secrets)
+
+<details>
+<summary>Click to reveal commands</summary>
+
+```sh
+echo -n "windows-2019" | gcloud secrets create image_factory-image_family --replication-policy="automatic" --data-file=-
+
+echo -n "golden-windows" | gcloud secrets create image_factory-image_name --replication-policy="automatic" --data-file=-
+
+echo -n "n1-standard-1" | gcloud secrets create image_factory-machine_type --replication-policy="automatic" --data-file=-
+
+echo -n "australia-southeast1" | gcloud secrets create image_factory-region --replication-policy="automatic" --data-file=-
```

Comment:
:D

We usually use `us-central1` as default

Reply:
done